codeforamerica/vita-min
# Fyst 1463 collect spouse date of death from qss filers reqd for e certification

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1463
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Added a new page at the beginning of the NC flow asking for the year of death of the QW filer's spouse.
## How to test?
- Start the NC flow as a non-QW filer and verify that the page doesn't show up
- Start the NC flow as Pork E & verify that the page shows up, that it shows appropriate errors if submitted empty, and that appropriate output shows up in our filing output
## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/2eff34af-fbbf-4ca8-a456-31bc797c1aef)
